app/scripts/constants/sentry-state.ts

```diff
@@ -0,0 +1,400 @@
+import { AllProperties } from '../../../shared/modules/object.utils';
+
+export const MMI_SENTRY_BACKGROUND_STATE = {
```

Comment:
I copied this over again from the constants in `setupSentry.js` on the v12.0.6 branch, so that they remain aligned.